src/nwb_benchmarks/command_line_interface.py

```diff
@@ -26,6 +28,19 @@ def main() -> None:
 
     default_asv_machine_file_path = pathlib.Path.home() / ".asv-machine.json"
     if command == "run":
+        asv_root = pathlib.Path(__file__).parent.parent.parent / ".asv"
+        asv_root.mkdir(exist_ok=True)
+        intermediate_results_folder = asv_root / "intermediate_results"
+
+        if intermediate_results_folder.exists():
```

Comment:
Also including this quality of life tidbit

Annoying otherwise to have a corrupt intermediate state only to learn about it after benchmarks complete

Reply:
Looks good